### PR TITLE
feat: added toasts and snackbar

### PR DESCRIPTION
**Pull request by aryan-13**
-Added 2 variations of snackbar
-added 2 variations of toasts

toast

### CODE REVIEW

File: components/snackbar.css [hunk 1]
```diff
@@ -0,0 +1,90 @@
...
+/* Snackbar buttons */
+.snackbar-btn-container {
+	justify-items: end;
+	display: flex;
+	margin-left: auto;
+}
+.snackbar-btn {
+	color: white;
+    text-align: center;
+background-color: inherit;
```
Comment (junaidshaikh-js) [inline components/snackbar.css]: format your code before commit

**Review by junaidshaikh-js — COMMENTED** [review]
Use code formatting tool.

File: components/snackbar.css (hunk 1 above)
Comment (aryan-13) [reply components/snackbar.css]: Done!

**Review by raghav2110 — COMMENTED** [review]
As suggested in previous comment can format code for better understanding. Overall looks good to merge.

**Pull request merged**